manage_breast_screening/core/jinja2/layout-app.jinja

```diff
@@ -7,6 +7,8 @@
   <script type="module" src="{{ static('js/bundle.js' )}}"></script>
 {% endblock %}
 
+{% block pageTitle %}{% if form and form.errors %}Error: {% endif %}{% if page_title %}{{page_title}} - {% endif %}Manage breast screening - NHS{% endblock %}
```

Comment:
Can you double check if this should be a hyphen or an en-dash?

```suggestion
{% block pageTitle %}{% if form and form.errors %}Error: {% endif %}{% if page_title %}{{page_title}} – {% endif %}Manage breast screening - NHS{% endblock %}
```

Reply:
En dash! Note @colinrotherham's suggestion seems to change one of the two.

Reply:
Well spotted! Didn't know if en-dashes went the way of curly quotes